pom.xml

```diff
@@ -48,6 +48,7 @@
 		<module>web</module>
 		<module>wildfly</module>
 		<module>testsuite</module>
+                <module>sso</module>
```

Comment:
Ah, I removed this once already. :-)
There is a reason this is just being run in a "sso" profile -- so you don't drag all the stuff in.